mail_activity_done/hooks.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018-22 ForgeFlow <http://www.forgeflow.com>
+# Copyright 2018 Odoo, S.A.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import Command, fields
+
+from odoo.addons.mail.models.mail_activity import MailActivity
+
+
+def pre_init_hook(cr):
+    """The objective of this hook is to default to false all values of field
+    'done' of mail.activity
+    """
+    cr.execute(
+        """SELECT column_name
+    FROM information_schema.columns
+    WHERE table_name='mail_activity' AND
+    column_name='done'"""
+    )
+    if not cr.fetchone():
+        cr.execute(
+            """
+            ALTER TABLE mail_activity ADD COLUMN done boolean;
+            """
+        )
+
+    cr.execute(
+        """
+        UPDATE mail_activity
+        SET done = False
+        """
+    )
+
+
+def post_load_hook():
+    def _new_action_done(self, feedback=False, attachment_ids=None):
```

Comment:
this monkey patch is potentially harmful and we don't need it. Would you mind cherry picking https://github.com/OCA/social/pull/1011 into your migration?